tests/utilities/test_utilities.py

```diff
@@ -361,6 +362,35 @@ def test_s_enhance_3D_no_obs(s_enhance):
                                                               f].mean())
 
 
+def test_t_coarsen():
```

Comment:
good addition :)

Reply:
Haha yeah I cant believe we didnt have an explicit test for this previously! i was so nervous that this test would fail!

Reply:
lmao that would have been nuts